src/parser/jobs/ast/modules/Sect/Sect.tsx

```diff
@@ -166,14 +93,14 @@ export default class Sect extends Module {
 		/*
 			SUGGESTION: Pulled without Sect
 		*/
-		if (!this.gaveup && (this.pullWithoutSect || !this.activeSectId)) {
+		if (!this.gaveup && (this.pullWithoutSect || this.activeSectId === 0)) {
```

Comment:
Null check preferable here too, for clarity.

Reply:
updated for clarity